src/main/java/eu/europeana/metis/sandbox/service/workflow/HarvestServiceImpl.java

```diff
@@ -161,15 +161,9 @@ private RecordInfo harvestOaiRecords(String datasetId, OaiHarvestData oaiHarvest
 
       return recordInfo;
     } catch (HarvesterException | IOException e) {
-      LOGGER.error("Error harvesting OAI-PMH Record Header: {} with exception {}",
-          oaiHarvestData.getOaiIdentifier(), e);
-      RecordError recordErrorCreated = new RecordError(
-          "Error harvesting OAI-PMH Record Header:" + oaiHarvestData.getOaiIdentifier(),
-          e.getMessage());
-      recordErrors.add(recordErrorCreated);
```

Comment:
Could you explain how by removing this solves the problem? I thought that it would be necessary to save the errors for later in the data processing.
Also, since this `recordErrors` list is not being used for saving errors anymore, we can remove it

Reply:
The logic was moved inside the method ```saveErrorWhileHarvesting``` and it is stored  in the database.